clients/api-proxy/rne/index.ts

```diff
@@ -103,7 +104,7 @@ const mapToDomainObject = ({
       siren: personneMorale.siren,
       denomination: personneMorale.denomination,
       natureJuridique: personneMorale.natureJuridique,
-      role: personneMorale.role,
+      role: (personneMorale.role || '').toUpperCase(),
```

Comment:
In the case where it is :
INPI: Directeur Général
IG: DIRECTEUR GENERAL
Maybe it won't work ?

We could create an util for roles that also remove special characters :
export const formatRole = (role: string) => {
  if (!role) {
    return '';
  }
  return removeSpecialChars(role).toUpperCase();
};

Reply:
Also, should we apply this rule to the rechercheEntreprise client ?